### ISSUE TITLE
Getting started with PyTorch tutorial

### ISSUE THREAD

**Issue by iancze** [MPoL-dev/MPoL#26]
**Is your feature request related to a problem or opportunity? Please describe.**
A small, self-contained tutorial that introduces PyTorch to astronomers that may be unfamiliar with it.

**Describe the solution you'd like**
This would mostly follow the arc of other introductory tutorials (e.g., [PyTorch docs intro](https://pytorch.org/tutorials/beginner/basics/intro.html), [Stanford CS230](https://cs230.stanford.edu/blog/pytorch/)) introducing and discussing 
* Initialization of tensors
* Calculation of gradients
* The idea of gradient descent (without yet introducing the PyTorch optimizer object)

These could be built up around the idea of minimizing an, easily plotted, 1D function whose gradient can be caclulated analytically. This could be a polynomial with a clear global minimum and no local minima in the search space.

The main purpose is to assuage the fears of new users who are familiar with Python, might be interested in using MPoL, but would see the unfamiliar requirement of PyTorch as a potential barrier. The goal is to show users that using MPoL is just like writing Python. At the end of the tutorial, links to numerous other PyTorch introductory materials will be provided.

**Describe alternatives you've considered**
We've considered just linking out introductory material to existing PyTorch guides, but I think there is enough mismatch in scope and tone that many new MPoL users would find this disorienting (e.g., how does this apply to RML imaging?!?) and lose interest in MPoL. After landing on the documentation page, we want them to quickly understand that this RML package is accessible to normal (astronomical) Python users.

**Comment by iancze** [comment]
Closed by #60
